nixos/modules/services/audio/mopidy.nix

```diff
@@ -70,25 +70,25 @@ in {
 
   config = mkIf cfg.enable {
 
+    systemd.tmpfiles.rules = [
+      "d '${cfg.dataDir}' - mopidy mopidy - -"
```

Comment:
Is this recursive btw?

Reply:
No. I only added the set recursive "Z" option when someone explicitly requested/thought was required.